src/compiler/checker.ts

```diff
@@ -47578,13 +47578,13 @@ export function createTypeChecker(host: TypeCheckerHost): TypeChecker {
         }
     }
 
-    function getDiagnostics(sourceFile: SourceFile, ct: CancellationToken): Diagnostic[] {
+    function getDiagnostics(sourceFile: SourceFile, ct: CancellationToken, forceDtsEmit?: boolean): Diagnostic[] {

```

Comment:
Maybe rename the parameter to `skipDiagnostics` from here on in, since it's no longer really concerned with emit, per sey, at this point?